Packs/GoogleBigQuery/Integrations/GoogleBigQuery/GoogleBigQuery.py

```diff
@@ -100,7 +100,7 @@ def build_query_job_config(allow_large_results, default_dataset_string, destinat
 
 
 def convert_to_string_if_datetime(object_that_may_be_datetime):
-    if isinstance(object_that_may_be_datetime, datetime):
+    if isinstance(object_that_may_be_datetime, datetime) or isinstance(object_that_may_be_datetime, time):
```

Comment:
As mentioned above, substitue 'time' with the name you choose for the import.
```suggestion
    if isinstance(object_that_may_be_datetime, datetime | time):
```